Rewrite of Yuki punishment_edging

DiligentJackal opened the pull request:
Rewritten for tone, focusing on Yuki's stated kinks, and shortening the scene to be more in line with other DomSim content.

deviant-dev commented on ink/debug:
```diff
@@ -144,6 +144,7 @@
 
 
 
+/*
```
Hmm. It won't matter for accepting the changes you have here, but it's there a reason you commented out these sections? The didn't affect the scripts unless they are calls via the debug diverts above.

HeadphoneTaco replied:
Commented them out to make it look a little cleaner, I'll leave them alone next time

deviant-dev commented on ink/goddess/generic_responses:
```diff
@@ -1,52 +1,111 @@
+    Eyes on me, {player_name}. #133
+      /wait 1
 }
+/*Added in waits, as the lines felt rushed when transitioning*/
```
Good idea!

A side note: For comments, let's stick to `//` instead of `/* ... */`. I have a few things that Annalise the scripts and I haven't really taken `/*` into account. You can select multiple lines and comment/uncomment with `ctrl+/`

HeadphoneTaco replied:
Will do! I'll change any /* ... */ I find

deviant-dev commented on ink/goddess/generic_responses:
```diff
@@ -1,52 +1,111 @@
 
-/perform talk_thoughtful skeptical
-
+//TODO: Make # of cycle elements equal or change to cycle/shuffle
```
Any particular reason for this Todo? Or is this something I wrote from somewhere else?

HeadphoneTaco replied:
The # of elements in the 2 animation cycles in the Goddess_Payattention Knot were off by one, so the combinations would start drifting. I wasn't sure if that was an intentional thing, or made to help the game feel less scripted.

deviant-dev commented on ink/goddess/generic_responses:
```diff
@@ -1,52 +1,111 @@
-    - As I was saying... #138
-    - Now... Let's start over. #139
+
+    - /perform talk_consider happy 60
```
Great job breaking all these it into individual performances per line.

HeadphoneTaco replied:
Thanks! I've been expanding a lot of the code, to help with reading

deviant-dev commented on ink/goddess/intro:
```diff
@@ -10,98 +10,134 @@
-/scene bedroom.flash1
-/music goddess
-/ambient orgy
+  /scene bedroom.flash1
```
Doesn't affect anything, but are the extra spaces an accidentOr a formatting choice?

HeadphoneTaco replied:
I think it was an accident

deviant-dev commented on ink/goddess/intro:
```diff
@@ -10,98 +10,134 @@
+
+ { cycle:
+
+          - It's a simple question.
+           /perform react_nod_big disgusted 60
+           Nod your head for yes. #09
```
For future reference: Lines can't be separated out like this without re-editing the audio files and creating a new tag. It shows up as a missing line and shows "It's a simple question" as a subtitle because the line isn't tagged and doesn't have a VO file associated with it.

HeadphoneTaco replied:
Whoops. I'll not do that again.

deviant-dev commented on ink/goddess/intro:
```diff
@@ -10,98 +10,134 @@
    Oh how absolutely delusional you are! #23
-   /perform react_shrug_long
+
+   /perform react_shrug_long 65
```
Looks like there's a type on this line and the emotion got removed somehow. I got a warning while playing through "Could not find emotion '65' on character Goddess". :)

I removed the 65 as a quick cleanup.

HeadphoneTaco replied:
Huh, pretty sure I put an emotion there. Oh well, the skeptical being carried forward works just fine

HeadphoneTaco commented on ink/_web.ink:
```diff
@@ -67,20 +67,21 @@ INCLUDE promo/devil_girl
-
+//Uncommented lines 72-77
 // EXTERNAL isDebug()
-// EXTERNAL isWeb()
```
Okay, I'll not commit changes to _web.ink in the future

HeadphoneTaco commented on ink/yuki/intro:
```diff
@@ -12,30 +12,30 @@
 // /equip feather_duster
 
-/perform idle_pleased happy 75
+/perform idle_slave aroused 75
```
Ack, that was part of my testing when I was figuring out the _game.json. While I'm going through the Yuki intro again, I'll fix those.

HeadphoneTaco commented on ink/goddess/intro:
```diff
@@ -182,53 +217,48 @@ I bet you have no idea where you are, do you? #36
+  You may never find your way out again. #42
 
-/perform talk_thoughtful neutral
+  /perform do_air_quotes aroused 70
```
I see what you mean, the revert sounds good

HeadphoneTaco commented on ink/goddess/intro:
```diff
@@ -419,62 +447,60 @@ I want to see you present yourself more fully to me. #100
+  but you'll need much more training first. #110
 
-but you'll need much more training first. #110
+  /perform do_now_leave amused 60
```
No, it's not angry or aggressive, she throws her open palmed hand up to the side

DiligentJackal committed:
Rewrite of Yuki punishment_edging

Rewriting for tone, focusing on Yuki's stated kinks, and shortening the scene to be more in line with other DomSim content.

DiligentJackal committed:
More scene and pose clean-up

Starting to add puppy pose logic as I understand Ink better.

Adding breakpoints throughout the scene and adjusting more dialogue.

Removed extraneous bits that were making the scene very long, content could be shuffled to its own scene entirely, there's a lot of it. :)

DiligentJackal committed:
Further rewriting Yuki punishment/edging

More puppy pose logic, dialogue rewrites, improved flow of scene by actually using the Play pane to test it.
